rhel7/profiles/rhelh-stig.profile

```diff
@@ -422,7 +422,7 @@ selections:
     - auditd_data_retention_space_left
     - audit_rules_execution_setfiles
     - audit_rules_kernel_module_loading_finit
-    - install_mcafee_antivirus
+    - install_antivirus
```

Comment:
Rule install_antivirus is already present in this profile on line 319.

Reply:
Thanks! Fixed.